plugin_test.go

```diff
@@ -0,0 +1,68 @@
+package main_test
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin/plugintest"
+	keks "github.com/matterpoll/matterpoll"
```

Comment:
What does `keks` means?
I have never seen that word.

Reply:
:+1: That is a typo. :smile: Will fix it